setup.py

```
@@ -63,24 +104,17 @@ def load_module_from_path(module_name, path):
     raise ValueError(
         "ROCM support is enabled (USE_ROCM=True) but ROCM_HOME is not set or detected.")
 
-if not USE_ROCM and not CUDA_HOME:
+if USE_CUDA and not CUDA_HOME:
     raise ValueError(
-        "CUDA support is enabled by default (USE_ROCM=False) but CUDA_HOME is not set or detected.")
+        "CUDA support is enabled by default on linux if `USE_ROCM=False`," \
+        " but CUDA_HOME is not set or detected.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)